### ISSUE TITLE
`remark-lint-no-shell-dollars` seems not working

### ISSUE THREAD

**Issue by AlexWayfer** [remarkjs/remark-lint#230]
### Subject of the issue

I'm trying to use `remark-lint-no-shell-dollars` and can't get offenses.

### Your environment

*   **OS**: Arch Linux
*   **Packages**:
```json
"remark-cli": "^8.0.0",
"remark-lint-no-shell-dollars": "^2.0.0",
"remark-preset-lint-markdown-style-guide": "^3.0.0",
```
*   **Env**: Node v10.16.0, NPM 6.9.0, Yarn 1.22.4.

### Steps to reproduce

```yaml
# .remarkrc.yaml
plugins:
  - - remark-lint-no-shell-dollars
    - error
```

``````markdown
```shell
$ bundle install && yarn install
```

Start ElasticSearch:

```sh
$ docker-compose up -d
```
``````

### Expected behaviour

There are offenses.

### Actual behaviour

No offenses.

**Comment by wooorm** [comment]
Hmm, the [code for the rule](https://github.com/remarkjs/remark-lint/blob/master/packages/remark-lint-no-shell-dollars/index.js) is a bit relaxed: it doesn’t do anything if the code is a single line

It will warn if you have two lines: `$ foo\n$ bar`, but then again it won’t warn if you use dollars for input and no dollars for output: `$ foo\nbar`.

What would you suggest?

**Comment by AlexWayfer** [comment]
> Hmm, the [code for the rule](https://github.com/remarkjs/remark-lint/blob/master/packages/remark-lint-no-shell-dollars/index.js) is a bit relaxed: it doesn’t do anything if the code is a single line

This is not documented.

> What would you suggest?

`no-shell-dollars` included into `preset-lint-markdown-style-guide`, which based on https://cirosantilli.com/markdown-style-guide#dollar-signs-in-shell-code, and there is no text that single-line is OK. Output is OK, so if the rule detected a mix of dollars and non-dollars — we can leave it, but single lines… why?

**Comment by wooorm** [comment]
In that case, I believe changing:

https://github.com/remarkjs/remark-lint/blob/52c8590ab5e3b9f2576b829daa30dbe0deffc785/packages/remark-lint-no-shell-dollars/index.js#L88

to:

```js
      if (length === 0) {
```

Should do what you want. Are you interested in creating a PR?

**Comment by AlexWayfer** [comment]
> Are you interested in creating a PR?

Friendly reminder about #231.

**Comment by wooorm** [comment]
Thanks for the ping @AlexWayfer!
